cmd/admin-console.go

```diff
@@ -18,42 +18,18 @@
 package cmd
 
 import (
-	"context"
-	"fmt"
-	"strings"
-	"time"
-
-	"github.com/fatih/color"
 	"github.com/minio/cli"
-	json "github.com/minio/colorjson"
-	"github.com/minio/madmin-go"
-	"github.com/minio/mc/pkg/probe"
-	"github.com/minio/pkg/console"
 )
 
-const logTimeFormat string = "15:04:05 MST 01/02/2006"
-
-var adminConsoleFlags = []cli.Flag{
-	cli.IntFlag{
-		Name:  "limit, l",
-		Usage: "show last n log entries",
-		Value: 10,
-	},
-	cli.StringFlag{
-		Name:  "type, t",
-		Usage: "list error logs by type. Valid options are '[minio, application, all]'",
-		Value: "all",
-	},
-}
-
 var adminConsoleCmd = cli.Command{
 	Name:            "console",
 	Usage:           "show console logs for MinIO server",
 	Action:          mainAdminConsole,
 	OnUsageError:    onUsageError,
 	Before:          setGlobalsFromContext,
-	Flags:           append(adminConsoleFlags, globalFlags...),
+	Flags:           globalFlags,
 	HideHelpCommand: true,
+	Hidden:          true, // deprecated June 2022
```

Comment:
do we need the comment here?

Reply:
Many a places i saw the date in comment so i did it.

Reply:
> Many a places i saw the date in comment so i did it.

This just makes it easier to know when we can remove deprecated things, like here after one year from now, it will be easier to decide to remove this deprecated code